mempool/mempool_test.go

```diff
@@ -22,6 +24,17 @@ func TestMempool(t *testing.T) {
 		require.Error(t, err)
 	})
 
+	t.Run("deposit transaction", func(t *testing.T) {
+		_, depositTx, _ := testutils.GenerateEthTxs(t)
+		depositTxBytes, err := depositTx.MarshalBinary()
+		require.NoError(t, err)
+
+		cosmosTxs, err := monomer.AdaptPayloadTxsToCosmosTxs([]hexutil.Bytes{depositTxBytes}, nil, "")
+		require.NoError(t, err)
+
+		require.Error(t, pool.Enqueue(cosmosTxs[0]))
```

Comment:
Can we use `ErrorContains` here, and test for the exact error string? It's generally not a great approach, but I think it's warranted here since we want to make sure we're hitting the correct error :)

Reply:
ErrorIs will require us to export the error, which we don't want to do.

Reply:
[Change ErrorIs to ErrorContains](https://github.com/polymerdao/monomer/pull/211/commits/55be3960045327498979d4a7b634a99d53c35dc0)